packages/app/app/components/HelpModal/Contributors/index.tsx

```diff
@@ -16,6 +31,8 @@ const Contributors = ({
     return <div className={styles.contributors_error}>There was an error loading contributors from Github</div>;
   }
 
+  const handleGithubClick = (link: string) => link && electron.shell.openExternal(link);
```

Comment:
Correct me if I'm wrong, but this needs an argument, but will be called with an event argument on click.
This name is misleading, since it opens any link, not just Github links.